### PR TITLE
feat: Add Files on Onboarding :sparkles:

### PR DESCRIPTION
**Pull request by kosssi**
The feature adds a page during onboarding to inform about the rights of files in the application.

#### Checklist

Before merging this PR, the following things must have been done:

- [x] Faithful integration of the mockups at all screen sizes
- [x] Tested on supported browsers (iOS & Android)
- [x] Localized in english and french
- [x] All changes have test coverage
- [ ] Updated README, if necessary

### CODE REVIEW

File: mobile/src/components/OnBoarding.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import classNames from 'classnames'
+
+import { translate } from '../../../src/lib/I18n'
+import Breadcrumb from './Breadcrumb'
+import styles from '../styles/onboarding'
+
+export const OnBoarding = ({ t, onActivate, onSkip, nameStep, currentStep, totalSteps }) =>
```
Comment (enguerran) [inline mobile/src/components/OnBoarding.jsx]: `nameStep` means the `step of the name`. I think you meant `stepName`.

File: mobile/src/components/OnBoarding.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      >
+        {onSkip && t('mobile.onboarding.step.button')}
+        {!onSkip && t('mobile.onboarding.step.next')}
+      </button>
+      <Breadcrumb currentStep={currentStep} totalSteps={totalSteps} />
+    </footer>
+  </div>
+)
+
+export default translate()(OnBoarding)
```
Comment (enguerran) [inline mobile/src/components/OnBoarding.jsx]: It worth to add `propTypes` to this component.

File: mobile/src/containers/onboarding/Analytics.jsx [hunk 3]
```diff
@@ -1,39 +1,11 @@
...
-        role='button'
-        className={'coz-btn coz-btn--regular'}
-        onClick={onActivate}
-      >
-        {t('mobile.onboarding.step.button')}
-      </button>
-      <Breadcrumb currentStep={2} totalSteps={2} />
-    </footer>
-  </div>
+  <OnBoarding onActivate={onActivate} onSkip={onSkip} nameStep='analytics' currentStep={3} totalSteps={3} />
```
Comment (enguerran) [inline mobile/src/containers/onboarding/Analytics.jsx]: I think it'll be hard to maintain that `currentStep` magic number.
The authority (the entity that knows what are the current step and the steps total) must be the parent component.

File: mobile/src/containers/onboarding/BackupPhotosVideos.jsx [hunk 4]
```diff
@@ -1,39 +1,11 @@
...
-        role='button'
-        className={'coz-btn coz-btn--regular'}
-        onClick={onActivate}
-      >
-        {t('mobile.onboarding.step.button')}
-      </button>
-      <Breadcrumb currentStep={1} totalSteps={2} />
-    </footer>
-  </div>
+  <OnBoarding onActivate={onActivate} onSkip={onSkip} nameStep='photos' currentStep={2} totalSteps={3} />
```
Comment (enguerran) [inline mobile/src/containers/onboarding/BackupPhotosVideos.jsx]: I think it'll be hard to maintain that `currentStep` magic number.
The authority (the entity that knows what are the current step and the steps total) must be the parent component.

File: mobile/src/containers/onboarding/Files.jsx [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import OnBoarding from '../../components/OnBoarding'
+
+export const Files = ({ onActivate }) =>
+(
+  <OnBoarding onActivate={onActivate} nameStep='files' currentStep={1} totalSteps={3} />
```
Comment (enguerran) [inline mobile/src/containers/onboarding/Files.jsx]: I think it'll be hard to maintain that `currentStep` magic number.
The authority (the entity that knows what are the current step and the steps total) must be the parent component.

File: mobile/src/containers/onboarding/Analytics.jsx (hunk 3 above)
Comment (kosssi) [inline mobile/src/containers/onboarding/Analytics.jsx]: I'm agree with you but It's another refactoring ;) I've decided to report it to another time.

File: mobile/src/components/OnBoarding.jsx (hunk 2 above)
Comment (kosssi) [inline mobile/src/components/OnBoarding.jsx]: I don't understand

File: mobile/src/containers/onboarding/Analytics.jsx (hunk 3 above)
Comment (goldoraf) [inline mobile/src/containers/onboarding/Analytics.jsx]: Yeah, we talked about that yesterday, another refactoring will follow

Comment (enguerran) [inline mobile/src/containers/onboarding/Analytics.jsx]: okay, we have a deal 🤝

File: mobile/src/components/OnBoarding.jsx (hunk 2 above)
Comment (enguerran) [inline mobile/src/components/OnBoarding.jsx]: see https://facebook.github.io/react/docs/typechecking-with-proptypes.html#react.proptypes

**Pull request merged**